cmd/csi-attacher/main.go

```diff
@@ -147,7 +147,7 @@ func main() {
 			pvLister := factory.Core().V1().PersistentVolumes().Lister()
 			nodeLister := factory.Core().V1().Nodes().Lister()
 			vaLister := factory.Storage().V1beta1().VolumeAttachments().Lister()
```

Comment:
Hm we should use v1 VolumeAttachment too.

Reply:
I can create another PR to address that...